mk/dev.mk

```diff
@@ -7,7 +7,7 @@ GIT_TAG = $(word 2, $(BUILD_INFO))
 GIT_COMMIT = $(word 3, $(BUILD_INFO))
 BUILD_DATE = $(word 4, $(BUILD_INFO))
 CI_TOOLS_VERSION = $(word 5, $(BUILD_INFO))
-ENVOY_VERSION ?= 1.30.4-k.1
+ENVOY_VERSION ?= 1.31.0
```

Comment:
Does it contain DNS patch?

Reply:
I think it has https://github.com/kumahq/envoy-builds/pull/105/files